orchestrator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "orchestator",
+  "version": "1.0.0",
+  "description": "",
+  "main": "bootstrap.ts",
+  "scripts": {
+    "build": "tsc --build",
+    "start": "ts-node index.ts"
+  },
+  "keywords": [],
+  "author": "Iryna Kopyl",
+  "license": "ISC",
+  "devDependencies": {
+    "@eslint/js": "^9.4.0",
+    "@types/eslint__js": "^8.42.3",
+    "@types/node": "^20.12.12",
+    "@types/uuid": "^9.0.8",
+    "@types/express": "^4.17.21",
+    "dotenv": "^16.4.5",
+    "eslint": "^8.57.0",
+    "express": "^4.19.2",
+    "express-validator": "^7.1.0",
+    "globals": "^15.4.0",
+    "kafkajs": "^2.2.4",
```

Comment:
It's strange that kafkajs, dotenv, express, uuid are in devDependencies. I doubt that without these libs the app will work)
For test, you can do `npm ci --production` and see if the project will work.
This comment applies to all microservices